README.md

```diff
@@ -105,7 +105,7 @@ In addition to open sourcing of the SDK, Microsoft has opened up the conceptual
 Build Instructions
 ==================
 
-This project uses the `project.json` for the project build system. Please see the [.NET Core SDK](https://www.microsoft.com/net/core#windows) for more information on how to use this project type.
+This project uses the new csproj format and the release versions of the tooling in Visual Studio 2017. For more info on how to use this project type to build your project, see the release notes for Visual Studio 2017: https://www.visualstudio.com/en-us/news/releasenotes/vs2017-relnotes#dotnetcore
```

Comment:
This is somewhat in contradiction to what is state below. I would list the projects that are listed below, as VS 2017 is not the only supported scenario. Once this is there, you can remove the details below starting with 'such as Visual Studio...'